docs/06-concepts/11-authentication/05-custom-overrides.md

```diff
@@ -177,3 +177,18 @@ var token = await client.authenticationKeyManager?.get();
 ```
 
 The above example retrieves the token from the key manager and stores it in the `token` variable.
+
+### Advanced authentication schemes
+
+By default Serverpod will pass the authentication token from client to server in accordance with the HTTP "authorization" header standard with the "basic" scheme name and encoding. This is securely transferred as the connection is TLS encrypted.
+
+The default implementation encodes and wraps the user-provided token in a "basic" scheme which is automatically unwrapped on the server side before being handed to the user-provided authentication handler described above. *In other words the default transport implementation is "invisible" to user code.*
```

Comment:
Question: Nothing wrong with using italics to accentuate parts, but I don't think this pattern is used elsewhere in the docs? I think consistency and coherence is very important in docs so that's why I'm wondering.

Usually I think if some parts needs to be extra highlighted we do an "info" or "attention" markdown block

Reply:
Well if we don't do it elsewhere then I agree we shouldn't here. But to me it doesn't quite fit to be a distinct 'info' block. I suggest just having it in its own paragraph, that seems appropriate emphasis

Reply:
Agree!